src/store/messaging.store.ts

```diff
@@ -237,7 +237,13 @@ export const useMessagingStore = create<MessagingState>((set, g) => {
         // You might want to update UI or state here
       },
       onHandshakeCompleted: (conversation, contact) => {
-        console.log("Handshake completed:", conversation);
+        set((s) => ({
+          oneOnOneConversations: s.oneOnOneConversations.map((oooc) =>
+            oooc.conversation.id === conversation.id
+              ? { ...oooc, conversation }
+              : oooc
+          ),
+        }));
       },
       onHandshakeExpired: (conversation, contact) => {
         console.log("Handshake expired:", conversation);
```

Comment:
I believe all of these events operations on the conversation manager should be removed, and should be handled by the caller for better clarity, what do you think?

Reply:
In fact, I wanted to remove all of that in an upcoming PR (still not done), but I think we can do that in this one if you feel it's ok